README.md

```diff
@@ -575,24 +537,23 @@ end
 module_function :score_page_hits
 ```
 
-It would have to be called by some code (definitely an action) which did the
-dirty work of talking to ActiveRecord. But it itself is pure.
+It would have to be called by an action already having done the dirty work of
+talking to ActiveRecord.
 
 **One of the chief ways of refactoring using Straitjacket is moving complex,
 side-effecting code "outward" into calling code, which at the outermost level is
 inevitably an action.**
 
-It is thus ironic that Straitjacket is implemented as a mixin. If you think
-about it, mixing in a module *is actually a state mutation* of the Ruby runtime.
-The very thing enabling our functional style of programming can never be "good".
-["Ceci n'est pas une pipe."](https://en.wikipedia.org/wiki/The_Treachery_of_Images)
-, indeed.
+It is a bit ironic that Straitjacket is implemented as a mixin. Really, mixins
+are actually side effects affecting the Ruby runtime itself. The very thing
+enabling our functional style of programming can never be "Good".
+["Ceci n'est pas une pipe."](https://en.wikipedia.org/wiki/The_Treachery_of_Images).
 
 In short:
 
 * Aspire to write Good code.
 * Aspire to make Bad code Good.
-* Feel free to use Ugly code anywhere, provided it doesn't break constraints.
+* Use Ugly code anywhere, provided it doesn't violate Straitjacket constraints.
```

Comment:
Shouldn't this be "use ugly code anyway, provided it doesn't adhere to Straitjacket constraints"?